docs/TRANSLATIONS.md

```diff
@@ -60,7 +60,7 @@ Check `tools/python/strings_utils.py -h` to see all of them.
 
 To check consistency of types_strings.txt with categories.txt run:
```

Comment:
But the script operates with `string.txt`...

Could you please add a comment to the script describing what exactly does it check?
I've tried to make some changes to the UI search categories in categories.txt but couldn't make the script fail.

Reply:
Added a comment in the script and updated the documentation.